packages/pwa-kit-react-sdk/src/ssr/universal/utils.ts

```diff
@@ -34,12 +34,15 @@ type GetAssetUrlOptions = {
  */
 export const getAssetUrl = (path: string) => {
     /* istanbul ignore next */
-    const publicPath = onClient
+    let publicPath = onClient
         ? // @ts-ignore
           `${window.Progressive.buildOrigin as string}`
-        : `${bundleBasePath}/${process.env.BUNDLE_ID || 'development'}/`
+        : `${bundleBasePath}/${process.env.BUNDLE_ID || 'development'}`
 
-    return path ? `${publicPath}${path}` : publicPath
+    // Normalize the public path by removing the trailing slash
+    publicPath = publicPath.replace(/\/$/, '')
```

Comment:
Do we want to also add the check we do to normalize leading slashes in getStaticAssetUrl into getAssetUrl?

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/01617982e298b3b75386cd8c4a62b833bc7905e1/packages/pwa-kit-react-sdk/src/ssr/universal/utils.ts#L74-L75